newIDE/app/src/Export/ExportDialog/ExportLauncher.js

```diff
@@ -330,6 +331,14 @@ export default class ExportLauncher extends Component<Props, State> {
     }));
   };
 
+  _getWebBuildThumbnailUrl = (buildId: string) => {
+    const project = this.props.project;
+    if (!project) {
+      return undefined;
+    }
+    return getWebBuildThumbnailUrl(project, buildId);
+  };
```

Comment:
The whole function can be simplified to `(buildId: string) => this.props.project && getWebBuildThumbnailUrl(this.props.project, buildId);`. IMO too verbose code takes too much time to read and is therefore less clean.

Reply:
Thanks, I made the change.

Reply:
This function `_getWebBuildThumbnailUrl` seems unused?